tests/unit/instructionAPI/syscall-x86.cpp

```diff
@@ -110,31 +110,12 @@ bool run_64()  {
   return true;
 }
 
-void usage(char const* prgname) {
-  std::cerr << "Usage: " << prgname << " [32|64]\n";
-}
-
-int main(int argc, char **argv) {
-  // Convention for CTest
-  constexpr int PASS =  0;
-  constexpr int FAIL =  1;
+int main() {
+  bool ok = run_32();
 
-  if(argc != 2) {
-    usage(argv[0]);
-    return FAIL;
+  if(!run_64()) {
```

Comment:
So this means even if we fail run_32, we still going to run run_64 anyway?

Reply:
Yeah. That's how the other tests I've written subsequently work. There's no need to fail early.